0055-rrna/sbatch.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+. ../common.sh
+
+task=$1
+log=$logDir/sbatch.log
+
+# NOTE!! The following must have the identical value set and used in rrna.sh
+out=$task.done
```

Comment:
This still differs from the value in `rrna.sh`, doesn't it?

Reply:
Ah yes. I got confused and thought it wasn't necessary after all. Got it now, done. I also got rid of $outUncompressed in rrna.sh, as there is never a need to compress the outfile.